discord/ext/commands/context.py

```diff
@@ -133,17 +183,9 @@ async def invoke(self, command, /, *args, **kwargs):
         TypeError
             The command argument to invoke is missing.
         """
-        arguments = []
-        if command.cog is not None:
-            arguments.append(command.cog)
-
-        arguments.append(self)
-        arguments.extend(args)
+        return await command(self, *args, **kwargs)
 
-        ret = await command.callback(*arguments, **kwargs)
-        return ret
```

Comment:
Simplfies the function, would you consider this outside the scope?

Reply:
Yes